src/style.css

```diff
@@ -0,0 +1,94 @@
+/* Basic styling for the To Do List */
+body {
+  font-family: Arial, sans-serif;
+  margin: 0;
+  padding: 0;
+  background-color: #f4f4f4;
+}
+
+.list-container {
+  max-width: 500px;
+  margin: 0 auto;
+  background-color: #fff;
+  margin-top: 40px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
+}
+
+h1 {
+  text-align: center;
+  padding: 20px 0;
+}
+
+.input {
+  width: 95%;
+  margin: 0 auto;
+  padding: 10px;
+  background-color: #f1eeee;
+  border: none;
+  border-radius: 5px;
+  margin-bottom: 10px;
+}
+
+.list {
+  list-style: none;
+  padding: 0;
+}
+
+.list li {
+  padding: 10px;
+  border-bottom: 1px solid #ddd;
+  background-color: #fff;
+  position: relative;
+}
+
+.completed {
+  text-decoration: line-through;
+  margin-left: 40px;
+  color: #333;
+}
+
+ul li::before {
+  content: "";
+  position: absolute;
+  height: 20px;
+  width: 20px;
+  border-radius: 50%;
+  background-image: url(images/uncheck.png);
+  background-size: cover;
+  background-position: center;
+  top: 12px;
+  left: -20px;
+}
```

Comment:
🔘 Great job on adding a custom bullet for the checkbox! However, there are a couple of adjustments I recommend:
- [ ] [Optional] First, the checkbox is currently not clickable. Please update your code to ensure that the checkbox is functional and clickable.
- [ ] [Optional] Second, the current checkbox design doesn't quite match the design you're aiming for. I would suggest modifying the checkbox style to align with the design's specifications. Keep up the good work! 📝🔧🚀

| Your | Design |
| --- | --- |
|  ![image](https://github.com/Umuhan/to-do-list/assets/55840999/6a6dd63e-8574-4ed4-8147-5f0cf4cbf869)| ![image](https://github.com/Umuhan/to-do-list/assets/55840999/b77cd611-54b6-43c5-b8dd-d3b04a500999) |